tests/latency_test/udp_lat.c

```diff
@@ -1129,7 +1138,12 @@ vma_recv_callback_retval_t myapp_vma_recv_pkt_filter_callback(
 
 	int recvsize     = iov[0].iov_len;
 	uint8_t* recvbuf = iov[0].iov_base;
-
+	if (user_params.enable_hw_time && !vma_info->hw_time_stamp) {
+		log_err("hw_timstamp is enables but hw not found");
+	}
+	if (!user_params.enable_hw_time && vma_info->hw_time_stamp) {
+		log_err("hw_timstamp is disabled but hw found");
```

Comment:
Sorry, by bad - had I noticed this is only a test I wouldn't have made the remark.